src/js/components/ServicesTable.js

```diff
@@ -118,23 +115,25 @@ var ServicesTable = React.createClass({
       );
     }
 
-    if (service instanceof Framework) {
-      // Get framework image/icon
-      itemImage = (
-        <span
-          className="icon icon-small icon-image-container icon-app-container icon-margin-right">
-          <img src={service.getImages()['icon-small']}/>
-        </span>
-      );
-    }
+    // Get service image/icon
+    return (
+      <span
+        className="icon icon-small icon-image-container icon-app-container icon-margin-right">
+        <img src={service.getImages()['icon-small']}/>
+      </span>
+    );
+  },
```

Comment:
I like the simplification a lot – but I have one question: shouldn't the `ServiceTree` struct also implement a `getImages()` method so that we can get rid of the `instanceof` check completely here?